commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/CommercePlacedOrderItemClayTable.java

```diff
@@ -139,10 +145,20 @@ public String getId() {
 				Locale locale = themeDisplay.getLocale();
 
 				String price = unitPriceMoney.format(locale);
-				String promoPrice = promoPriceMoney.format(locale);
 				String discount = discountAmountMoney.format(locale);
 				String total = finalPriceMoney.format(locale);
 
+				String promoPrice = StringPool.BLANK;
+
+				BigDecimal promoPriceValue = promoPriceMoney.getPrice();
+
+				if ((promoPriceMoney != null) &&
```

Comment:
Good point, Its mixed in the local service also. I'll change the CommerceMoney ones to better reflect the method thats used to get them. I.E. `CommerceMoney unitPromoPrice = commerceProductPrice.getUnitPromoPrice()`. Maybe the BigDecimal should just be `promoPrice` and the string should be `formattedPromoPrice` as it will have the formatted value `$1.00`

Reply:
It seems like in CommerceProductPrice the methods should have `Money` on the end of them, as they return a CommerceMoney right? Like we have with CommerceOrderItem